python/tk_multi_loader/banner.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+from sgtk.platform.qt import QtCore, QtGui
+import time
+
+
+class Banner(QtGui.QLabel):
+    """
+    Banner that can be shown and then dismissed with an animation. Will always
+    be shown for at least 3 seconds even if a request to hide it is done before
+    the time is up. The banner will always be displayed at the top of the parent
+    widget's window.
+    """
+
+    # Height of the widget.
+    _HEIGHT = 32
+
+    def __init__(self, parent):
+        """
+        :param parent: Parent widget.
+        """
+        super(Banner, self).__init__(parent)
+
+        # Sets the style sheet for the widget.
+        self.setStyleSheet("""
```

Comment:
The style sheet applies to the area under the builtin info banner at the top of everything Toolkit app. However, the banner is a child of the dialog itself, so the CSS doesn't apply.